homeassistant/components/fastdotcom/sensor.py

```diff
@@ -8,21 +8,46 @@
     SensorEntity,
     SensorStateClass,
 )
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import UnitOfDataRate
-from homeassistant.core import HomeAssistant, callback
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN, HomeAssistant, callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
 from homeassistant.helpers.restore_state import RestoreEntity
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-from . import DATA_UPDATED, DOMAIN as FASTDOTCOM_DOMAIN
+from .const import DATA_UPDATED, DOMAIN as FASTDOTCOM_DOMAIN
 
 
 async def async_setup_platform(
     hass: HomeAssistant,
     config: ConfigType,
     async_add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up the Fast.com sensor."""
+    async_create_issue(
```

Comment:
We should have an import here and the issue should be created in config flow depending on outcome. See ADR-21

Reply:
Can you clarify on this a bit please? I see tons of examples where the create_issue for the deprecation is stored in sensor.py/light.py, etc. What did I miss here? :)

Reply:
https://github.com/home-assistant/architecture/blob/master/adr/0021-YAML-integration-configuration-deprecation-policy.md
There are some PR's open so look at others now doing same thing as you do

Reply:
Got it - feedback has been processed.